.github/ISSUE_TEMPLATE/improve-existing-docs.yaml

```diff
@@ -0,0 +1,49 @@
+name: Improve existing content
+description: Make a suggestion to improve the content in an existing article
+title: '[What I really want the title to be]'
```

Comment:
I don't think that `title` is a required field, is it? Can we just leave this out?

Reply:
Yea I agree! I had the same question while writing the other two forms as well. Can we leave it out in the other two forms as well?

Reply:
Yes.

Reply:
Deleted the `title` key